proxy/contracts/test/SkaleVerifierMock.sol

```diff
@@ -27,7 +27,22 @@ import "../schain/bls/FieldOperations.sol";
 import "./PrecompiledMock.sol";
 
 
-contract SkaleVerifierMock {
+interface ISkaleVerifierMock {
```

Comment:
I guess it should be from skale-manager-interfaces

Reply:
Hm... It's a good idea!

Reply:
But it's blocked by SKALE-4466.